### PR TITLE
refactor eval_semantic_segmentation

### PR DESCRIPTION
**Pull request by knorth55**
split from #869 
refactor eval_semantic_segmentation
use same format as `eval_detection`

### CODE REVIEW

**Comment by yuyu2172** [comment]
pfnCI,test this please

**Comment by yuyu2172** [comment]
pfnCI,test this please

**Comment by yuyu2172** [comment]
Can you use `batchsize` here?
https://github.com/chainer/chainercv/blob/0557b8abe6df2bae50bbda83478762ab96ec25ba/examples/semantic_segmentation/eval_semantic_segmentation_multi.py#L40

After the change, the code should look like here https://github.com/chainer/chainercv/blob/master/examples/detection/eval_detection_multi.py#L39

File: examples/semantic_segmentation/eval_semantic_segmentation.py [hunk 1]
```diff
@@ -37,81 +45,73 @@ def get_dataset_and_model(dataset_name, model_name, pretrained_model,
...
-        input_size)
+        args.batchsize, args.input_size)
 
     if args.gpu >= 0:
         chainer.cuda.get_device_from_id(args.gpu).use()
         model.to_gpu()
 
-    it = iterators.SerialIterator(
+    iterator = iterators.SerialIterator(
         dataset, 1, repeat=False, shuffle=False)
```
Comment (yuyu2172) [inline examples/semantic_segmentation/eval_semantic_segmentation.py]: Please use batchsize

Comment (knorth55) [reply examples/semantic_segmentation/eval_semantic_segmentation.py]: updated

**Comment by knorth55** [comment]
Thanks, I updated.

**Pull request merged**